src/cartesian/Bar.tsx

```diff
@@ -115,6 +121,7 @@ export class Bar extends PureComponent<Props, State> {
     hide: false,
     data: [] as BarRectangleItem[],
     layout: 'vertical',
+    activeBar: true,
```

Comment:
I'm thinking this shouldn't be true by default https://github.com/recharts/recharts/issues/4103